spec/models/user_spec.rb

```diff
@@ -139,33 +139,6 @@ def generate_emailless_user
       expect(blocker).not_to have_interaction_blocked(only_posts)
       expect(blocker).to have_interaction_blocked(blockees.first)
     end
-
-    describe "content" do
```

Comment:
What's going on with this change?

Reply:
Oh, we don't use hidden_post_users or blocking_post_users anymore? Huh, strange.

Reply:
We weren't using that code! So I took a machete to it